### PR TITLE
Resolve @shopify/plugin-ngrok locally

### PR DESCRIPTION
**Pull request by amcaplan**
### WHY are these changes introduced?

Builds on main are broken from merging a version bump until we release it. This shouldn't happen.

### WHAT is this pull request doing?

We resolve packages locally in acceptance testing so we can create an app using the local packages before they're pushed to npm. This was broken on CI because we introduced a new plugin but didn't update its local source.

### How to test your changes?

CI

### Measuring impact

How do we know this change was effective? Please choose one:

- [x] n/a - this doesn't need measurement, e.g. a linting rule or a bug-fix
- [ ] Existing analytics will cater for this addition
- [ ] PR includes analytics changes to measure impact

### Checklist

- [x] I've considered possible cross-platform impacts (Mac, Linux, Windows)
- [x] I've considered possible [documentation](https://shopify.dev) changes

### CODE REVIEW

File: packages/create-app/src/utils/template/npm.ts [hunk 1]
```diff
@@ -54,8 +32,9 @@ export async function updateCLIDependencies({
     const dependencyOverrides = {
       '@shopify/cli': cliPath,
       '@shopify/app': appPath,
-      '@shopify/cli-kit': cliKitPath,
-      '@shopify/shopify-cli-extensions': extensionsCliPath,
+      '@shopify/cli-kit': await packagePath('cli-kit'),
+      '@shopify/shopify-cli-extensions': await packagePath('ui-extensions-cli'),
+      '@shopify/plugin-ngrok': await packagePath('plugin-ngrok'),
```
Comment (gonzaloriestra) [inline packages/create-app/src/utils/template/npm.ts:37]: What about creating constants above for all of them to keep consistency?

**Review by gonzaloriestra — APPROVED** [review]
Nice refactor!

File: packages/create-app/src/utils/template/npm.ts (hunk 1 above)
Comment (amcaplan) [reply packages/create-app/src/utils/template/npm.ts:37]: I debated internally about this. If you felt strongly enough to comment, I'll change it.

File: packages/create-app/src/utils/template/npm.ts [hunk 2]
```diff
@@ -21,30 +21,11 @@ export async function updateCLIDependencies({
...
-    const extensionsCliAbsolutePath = (await path.findUp('packages/ui-extensions-cli', {
-      type: 'directory',
-      cwd: moduleDirectory,
-    })) as string
-    const extensionsCliPath = `file:${extensionsCliAbsolutePath}`
+    const cliPath = await packagePath('cli-main')
+    const appPath = await packagePath('app')
+    const cliKitPath = await packagePath('cli-kit')
+    const uiExtensionsCliPath = await packagePath('ui-extensions-cli')
+    const pluginNgrokPath = await packagePath('plugin-ngrok')
```
Comment (pepicrft) [inline packages/create-app/src/utils/template/npm.ts:28]: Nice catch! If we continue adding packages to the monorepo we might want to evolve this implementation to get the list of packages using some globbing.

**Pull request merged**